source/vibe/web/rest.d

```diff
@@ -533,12 +533,21 @@ class RestInterfaceSettings {
 	*/
 	bool stripTrailingUnderscore = true;
 
+	/** By default, vibe.d will handle $(D HTTPStatusException)s and turn other
+	    exceptions into an Internal Server Error.
```

Comment:
I'd suggest to rewrite to 'this module will handle ...' (because when they pass through, it's still Vibe.d that handle them), and 'turn exception into JSON exception object with an error code of 500 and a statusMessage/statusDebugMessage [blabla]', or something of that vein, describing what is being sent on the wire.